articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,109 @@
+
+### Machine Learning and Artificial Intelligence
+Machine learning is a [subset](https://en.wikipedia.org/wiki/Subset) of AI that allows computer algorithms to make accurate data outcomes through interpretations.
+
+Machine learning allows the use the AI towards automating several functionalities used by machine learning. Besides this, the AI enhances neuroscience that emulates human intelligence, an aspect that machine learning lacks.
+
+Several [synthetic intelligence](https://www.wikiwand.com/en/Synthetic_intelligence) aspects and modern [virtual assistants](https://www.investopedia.com/terms/v/virtual-assistant.asp) are created to achieve this.
+
+Machine learning uses Artificial Intelligence (AI) techniques that assist in decision making. These techniques include the use of [conventional algorithms](http://wiki.cas.mcmaster.ca/index.php/Conventional_Encryption_Algorithms) used in performing mathematical computation and predictions.
+
+Structured data is presented in the form of columns and rows. Examples of structured data include strings, dates and numbers.
+
+Unstructured data information is stored as files. Examples include; images, word documents, email files, videos and audio files.
+
+Machine learning uses both structured and unstructured data. The use of Support Vector Machines (SVMs) and decision-trees are examples that use unstructured data to make interpretations.
+
+Conventional algorithms aids in achieving data mining and predictive analytics in a search. A conventional algorithm is an algorithm that aid in identifying, deciphering, and enciphering keys that are identical, and need to be kept secret.
+
+A conventional algorithm maintains secrets in a cryptographic environment. Conventional algorithm are also used to regulate algorithms used at undoing histogram modifications and distortions.
+
+Unlike machine learning, AI can use [unstructured data](https://www.datamation.com/big-data/structured-vs-unstructured-data.html) and still be able to function. Deep learning emulates human intelligence workings.
+Machine learning aids in the detection of objects, language translation, and decision making. Machine learning is also able to recognize speech and make prompt decisions.
+ AI enables technologies such as speech recognition without any human intervention.
+
+### Deep Learning
+Deep learning is a [subset](https://en.wikipedia.org/wiki/Subset) of machine learning. Deep learning uses deep [neural networks](https://www.investopedia.com/terms/n/neuralnetwork.asp#:~:text=A%20neural%20network%20is%20a,organic%20or%20artificial%20in%20nature.) that are built using neural networks. Neural networks can be used for both supervised and unsupervised learning tasks. Depending on the task, the algorithm has to work unsupervised, and vice versa, if there is supervision required.
```

Comment:
This sentence is very difficult to read:
"Deep learning uses deep neural networks that are built using neural networks. "

Reply:
Noted, I have provided enhanced more elaboration on the point.